pages/samvera/manager_guide/hyrax_2.1/create-works-2.1.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Creating a Work"
+permalink: create-works-2.1.html
+folder: samvera/manager_guide/create-works-2.1.md
+sidebar: home_sidebar
+a-z: ['Works - Creating a Work', 'Creating a Work']
+keywords: Best Practices, managers, repo mangers, hyrax administration
+tags: [user_resources]
+categories: How to use the Administration panel in hyrax
+version:
+  id: 'hyrax_2.1-stable'
+  versions:  
+  - label: 'Hyrax 1.0'
+    link:  'create-works-1.0.html'
+  - label: 'Hyrax 2.0'
+    link:  'create-works-2.0.html'
+  - label: 'Hyrax 2.1'
+    link:  'create-works-2.1.html'
+    selected: 'true'
+---
+# Creating a Work
+
+Works are items in the repository, containing files and metadata. Works are presented as one intellectual object. Example: A single dissertation (PDF and metadata), a dataset containing multiple files (spreadsheets, images, text files and metadata).
+
+A user must be logged in to create a Work. There are several ways to add a new work:
+
+- On the homepage, click "Share Your Work"
+- From the Dashboard, click on the "Works" sub-heading, click "Add new work"
+
+If your repository contains more than one work type, you will need to select the work type to proceed. Work types allow repository managers to create separate deposit forms based on the type of work, such as Dissertations, Images, or Datasets. Each work type might have specific metadata requirements or deposit instructions.
+
+![Add New Generic Work](images\screenshots\user-addgenericwork-2.png)
+
+There three tabs in the Add New Work interface: “Description”, “Files”, and “Relationships”. In order for the work to be deposited, a work must minimally have: the required description fields filled out, at least one file uploaded, and the deposit agreement checkbox checked. Once these required items are completed, the depositor must click “Save” in order to deposit the work. Navigating away from the deposit screen before clicking save causes user to have to redo deposit.
+
+## Description (required)
+The “Description” tab is for adding descriptive metadata. Fill out the fields marked “Required”. Clicking “Additional Fields” will show more metadata fields, which are optional. Customizing fields is possible by creating a new Work Type.
+
+## Files (required)
+![Add New Generic Work | Files](images\screenshots\user-addnewgenericwork-files-2.png)
+
+The “Files” tab is for uploading files, either locally or from a cloud service (e.g. DropBox, Google Drive, Box.com, etc.). You can browse local files to upload, or drag and drop files from your desktop. If your repository is configured to interact with a cloud service, users can click “Browse cloud files” and securely connect to their own Dropbox. They can then browse cloud files to upload. Files can also be removed from the deposit in the “Files” tab.
```

Comment:
Should we add "Repositories can be set up for metadata-only works, in this case files are not required."

Reply:
done